bucket/freeorion-test.json

```
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://freeorion.org/",
+    "description": "Turn-based space empire and galactic conquest (4X) game (Test Build)",
+    "version": "2021-10-26.9fd3aaa",
+    "license": "GPL-2.0",
+    "url": "https://sourceforge.net/projects/freeorion/files/FreeOrion/Test/FreeOrion_2021-10-26.9fd3aaa_Test_Win32_Setup.exe/download#/dl.zip",
+    "hash": "055bb5006017de058618b9650ec5181afb5e02ac3c8961766f548a841895c1f8",
+    "bin": [
+        [
+            "FreeOrion.exe",
+            "freeorion-test"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "FreeOrion.exe",
+            "FreeOrion (Test Build)"
+        ]
+    ],
+    "checkver": {
+        "url": "https://sourceforge.net/projects/freeorion/files/FreeOrion/Test/",
+        "regex": "FreeOrion_(?<year>[\\d]{4})-(?<month>[\\d]{2})-(?<day>[\\d]{2})\\.(?<commit>[a-z\\d]{7})_Test_Win32_Setup",
+        "replace": "${year}-${month}-${day}.${commit}"
+    },
+    "autoupdate": {
+        "url": "https://sourceforge.net/projects/freeorion/files/FreeOrion/Test/FreeOrion_$version_Test_Win32_Setup.exe/download#/dl.zip"
+    },
+    "notes": "User data are stored in C:\\Users\\[USERNAME]\\AppData\\Roaming\\FreeOrion"
```

We don't add a note for the user data path unless the user *must* drop something in there for the game to work (e.g. proprietary game data).

```suggestion
    }
```